numalogic/udfs/__main__.py

```diff
@@ -3,16 +3,24 @@
 import sys
 from typing import Final
 
-from numalogic._constants import DEFAULT_BASE_CONF_PATH, DEFAULT_APP_CONF_PATH, DEFAULT_METRICS_PORT
+from numaprom.monitoring import start_metrics_server
+
+from numalogic._constants import (
+    DEFAULT_BASE_CONF_PATH,
+    DEFAULT_APP_CONF_PATH,
+    DEFAULT_METRICS_PORT,
+    DEFAULT_METRICS_CONF_PATH,
+)
 from numalogic.connectors.redis import get_redis_client_from_conf
-from numalogic.monitoring import start_metrics_server
-from numalogic.udfs import load_pipeline_conf, UDFFactory, ServerFactory, set_logger
+from numalogic.udfs import load_pipeline_conf, UDFFactory, ServerFactory, set_logger, set_metrics
 
 LOGGER = logging.getLogger(__name__)
 
 BASE_CONF_FILE_PATH: Final[str] = os.getenv("BASE_CONF_PATH", default=DEFAULT_BASE_CONF_PATH)
 APP_CONF_FILE_PATH: Final[str] = os.getenv("APP_CONF_PATH", default=DEFAULT_APP_CONF_PATH)
 METRICS_PORT: Final[int] = int(os.getenv("METRICS_PORT", default=DEFAULT_METRICS_PORT))
+METRICS_ENABLED: Final[bool] = bool(os.getenv("METRICS_ENABLED", default="True"))
```

Comment:
Let's make it int. 1 for enabled, 0 for disabled. Because env var to native bool conversion can be tricky

Reply:
Sure!